client/my-sites/email/email-management/home/email-plan.jsx

```diff
@@ -336,86 +259,87 @@ class EmailPlan extends React.Component {
 
 			if ( isExpired( purchase ) ) {
 				return (
-					<VerticalNavItem onClick={ this.handleRenew } path="#">
+					<VerticalNavItem onClick={ handleRenew } path="#">
 						{ translate( 'Renew to add new mailboxes' ) }
 					</VerticalNavItem>
 				);
 			}
 
 			return (
-				<VerticalNavItem { ...this.getAddMailboxProps() }>
+				<VerticalNavItem { ...getAddMailboxProps() }>
 					{ translate( 'Add new mailboxes' ) }
 				</VerticalNavItem>
 			);
 		}
 
 		return (
-			<VerticalNavItem { ...this.getAddMailboxProps() }>
+			<VerticalNavItem { ...getAddMailboxProps() }>
 				{ translate( 'Add new email forwards' ) }
 			</VerticalNavItem>
 		);
 	}
 
-	render() {
-		const {
-			currentRoute,
-			domain,
-			selectedSite,
-			hasSubscription,
-			purchase,
-			isLoadingPurchase,
-		} = this.props;
+	const { domain, selectedSite, hasSubscription, purchase, isLoadingPurchase } = props;
 
-		const { isLoadingEmailAccounts } = this.state;
+	// Ensure we check for email forwarding additions and removals
+	const shouldQueryEmailForwards = shouldCheckForEmailForwards( domain );
 
-		// Ensure we check for email forwarding additions and removals
-		const queryForEmailForwards = this.shouldCheckForEmailForwards( domain );
+	const { data, isLoading } = useEmailAccountsQuery( props.selectedSite.ID, props.domain.name, {
+		retry: false,
+	} );
 
-		return (
-			<>
-				{ selectedSite && hasSubscription && <QuerySitePurchases siteId={ selectedSite.ID } /> }
-				{ queryForEmailForwards && <QueryEmailForwards domainName={ domain.name } /> }
-
-				<DocumentHead title={ titleCase( this.getHeaderText() ) } />
-
-				<HeaderCake onClick={ this.handleBack }>{ this.getHeaderText() }</HeaderCake>
-
-				<EmailPlanHeader
-					domain={ domain }
-					hasEmailSubscription={ hasSubscription }
-					isLoadingEmails={ isLoadingEmailAccounts }
-					isLoadingPurchase={ isLoadingPurchase }
-					purchase={ purchase }
-					selectedSite={ selectedSite }
-					emailAccount={ this.state.emailAccounts?.[ 0 ] }
-				/>
-
-				<EmailPlanMailboxesList
-					account={ this.getAccount() }
-					domain={ domain }
-					mailboxes={ this.getMailboxes() }
-					isLoadingEmails={ isLoadingEmailAccounts }
-				/>
-
-				<div className="email-plan__actions">
-					<VerticalNav>
-						{ this.renderAddNewMailboxesOrRenewNavItem() }
-
-						<UpgradeNavItem
```

Comment:
Follow-up comment: this item was not kept in the list after the refactor. 😬 I am adding it back now!